README.md

```diff
@@ -248,11 +248,11 @@ This value is **mandatory** for generation sitemap file, and you should explicit
 
 - Default: `1000 * 60 * 15` (15 Minutes)
 
-Defines how frequently should sitemap **routes** being updated (value in milliseconds).
+Defines how frequently sitemap **routes** should be updated (value in milliseconds).
 
 Please note that after each invalidation, `routes` will be evaluated again (see [routes declaration](#routes-declaration) section).
 
-This option is enable only for the nuxt **ssr** mode.
+This option is only available for nuxt in its **ssr** mode.
```

Comment:
```suggestion
This option is only available on **ssr** mode.
```

Reply:
Indeed it is probably not necessary to repeat nuxt everywhere. However in this case I would replace "on ssr mode" with "in ssr mode"

Reply:
```suggestion
This option is only available in **ssr** mode.
```